codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/SymbolVisitor.kt

```diff
@@ -399,13 +383,15 @@ fun symbolBuilder(shape: Shape?, rustType: RustType): Symbol.Builder {
 fun handleOptionality(symbol: Symbol, member: MemberShape, nullableIndex: NullableIndex, nullabilityCheckMode: CheckMode): Symbol =
     symbol.letIf(nullableIndex.isMemberNullable(member, nullabilityCheckMode)) { symbol.makeOptional() }
 
-// TODO(chore): Move this to a useful place
 private const val RUST_TYPE_KEY = "rusttype"
+private const val RUST_MODULE_KEY = "rustmodule"
 private const val SHAPE_KEY = "shape"
 private const val SYMBOL_DEFAULT = "symboldefault"
 private const val RENAMED_FROM_KEY = "renamedfrom"
 
 fun Symbol.Builder.rustType(rustType: RustType): Symbol.Builder = this.putProperty(RUST_TYPE_KEY, rustType)
+fun Symbol.Builder.module(module: RustModule.LeafModule): Symbol.Builder = this.putProperty(RUST_MODULE_KEY, module)
+fun Symbol.module(): RustModule.LeafModule = this.expectProperty(RUST_MODULE_KEY, RustModule.LeafModule::class.java)
```

Comment:
Are these properties read somewhere?

Reply:
yes—everywhere we need to compute the module for a system

Reply:
Yeah I even cited the line where we read them, my bad.